pom.xml

```diff
@@ -7,7 +7,7 @@
     <groupId>com.griddynamics.qa</groupId>
     <artifactId>sprimber-parent</artifactId>
     <packaging>pom</packaging>
-    <version>1.0.3-SNAPSHOT</version>
+    <version>1.0.5-SNAPSHOT</version>
```

Comment:
And where did we miss 1.0.4 version?

Reply:
As discussed - there were an issue with release process, that's why it's necessary to skip the version here